### PR TITLE
[Doc Improvement][Added mobile support notes][3348812]

### PR DESCRIPTION
**Pull request by JoshuaPartlow**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/69ecacec2a42a850bcd165b46983f013d6d95b64/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 97 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d3c209f9-5fe5-42c9-b46b-f7da7b674afc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/67fdb47dc0ca887c58c4200000fc3bab1a755117/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 97 | 5 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/83b27373-cbe3-4cf7-95ae-a0e074da5622) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/78fce804b2bdf0ccfe977c07ced6c3ae530ff45d/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/64fdf20f-bb69-4bf3-a049-8af4efb8b1e8) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/692f82e28fd3d6764f56fd9afa89705ae86eacbd/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d51e00f5-1994-429b-afa9-12ceec6576e6) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-deabira — DISMISSED** [review]

**Review by erikadoyle — APPROVED** [review]

File: msteams-platform/m365-apps/teamsjs-support-m365.md [hunk 1]
```diff
@@ -1921,6 +1941,11 @@ Preview. Namespace providing for profile-related functionality.
     </tbody>
 </table>
 
+> [!NOTE]
+> The following methods are not supported on mobile for Teams:
+>
+> `profile.showProfile`
```
Comment (v-sdhakshina) [inline msteams-platform/m365-apps/teamsjs-support-m365.md:1947]:
```suggestion

```

File: msteams-platform/m365-apps/teamsjs-support-m365.md [hunk 2]
```diff
@@ -2165,6 +2190,11 @@ Namespace to open a share dialog for web content. For more information, see [Sha
     </tbody>
 </table>
 
+> [!NOTE]
+> The following methods are not supported on mobile for Teams:
+>
+> `sharing.shareWebContent`
```
Comment (v-sdhakshina) [inline msteams-platform/m365-apps/teamsjs-support-m365.md:2196]:
```suggestion

```

File: msteams-platform/m365-apps/teamsjs-support-m365.md [hunk 3]
```diff
@@ -2287,6 +2317,11 @@ The earlier version of the capability for providing modal dialog (task module) s
     </tbody>
 </table>
 
+> [!NOTE]
+> The following methods are not supported on mobile for Teams:
+>
+> `task.getDefaultSizeIfNotProvided`
```
Comment (v-sdhakshina) [inline msteams-platform/m365-apps/teamsjs-support-m365.md:2323]:
```suggestion

```

File: msteams-platform/m365-apps/teamsjs-support-m365.md [hunk 4]
```diff
@@ -2470,11 +2511,16 @@ Preview. Contains functionality to allow web apps to store data in webview cache
     </tbody>
 </table>
 
+> [!NOTE]
+> The following methods are not supported on mobile for Teams:
+>
+> `webStorage.isWebStorageClearedOnUserLogOut`
```
Comment (v-sdhakshina) [inline msteams-platform/m365-apps/teamsjs-support-m365.md:2517]:
```suggestion

```

**Review by v-sdhakshina — CHANGES_REQUESTED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/da9d57d46a83f85e30fc15346b844def486402f1/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e4c1d877-9f2f-4559-8328-833547745720) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/29d021d6de255a7ed5cce209a15b4fbe0bf097d3/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3cb8311d-6275-4804-ab37-b5e468f0907c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/204bf0102ee7c194b7c668e8b481487c73549121/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/dabe5172-d1c1-442b-ae42-458c51874dbc) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0234dd31f58f0a6724f1ae518ef0db01644a16d8/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/74bca1db-d0d4-41a7-8755-9fb8e7df2d46) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/24f084f9992da3b634721baf6694b28e17b729e7/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7dcd096c-9751-45b4-8d8a-71c3d9dc8f7e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d1870baa4085ec94d63aaa4586a243189e8cc7cc/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ebbf7b10-ba64-4a6a-8c9f-2a2a755bfeae) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/a100a0c3e5ee660066d269377d7cac705492ab70/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/daa98f74-3298-4ac2-927e-7cb0e6ff84fa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1429017ed2f97ae578d0c1941544bf474925a87a/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8819e8e1-6bb8-4d07-8bba-1e263bd2aaae) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-sdhakshina — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/m365-apps/teamsjs-support-m365.md](https://github.com/MicrosoftDocs/msteams-docs/blob/49d656f401788bbcb313bd7c8c4c265dc1db85ce/msteams-platform%2Fm365-apps%2Fteamsjs-support-m365.md) | 96 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d57302ca-3973-4457-adc3-9b48cba68db9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Pull request merged**